docs/docs/user/references/defaults.md

```diff
@@ -0,0 +1,75 @@
+# Defaults
+
+KPOps has a very efficient way of dealing with repeating settings which manifests as [`defaults.yaml`](/resources/pipeline-defaults/defaults).
+
+This file provides the user with the power to set defaults for any and all components, thus omitting the need to repeat the same settings in [`pipeline.yaml`](/user/references/components).
```

Comment:
you're off to a good start here!

please also go into detail about the inheritance model, i.e. configuring a base component as shared config of its subcomponents.

You can use an example to further clarify this, e.g. source & sink connector with their base class `kafka-connect` containing the shared config for both

Reply:
Do you think the current examples are sufficient?